README.md

```diff
@@ -1 +1,144 @@
-# hello_microverse
\ No newline at end of file
+<a name="readme-top"></a>
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project">
+Hello , This is a HTML AND CSS simple project.</a>
+
+[Hello Microverse] is a simple buil up demo project.
+
+## 🛠 Built With 
+
+### Tech Stack 
+HTML
+CSS
+
+<!-- Features -->
+
+### Key Features 
+
+- **[Simple Site]**
+- **[h1 tag is used]**
+- **[Linter is used]**
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:You have to clone this project to your local machine
+
+### Setup
+
+Clone this repository to your desired folder:git clone git@github.com:Elhambasir/Hello-Microverse.git
+
+### Install
+
+Install this project with: Any software which you are comfortable with
+
+### Usage
+
+To run the project, execute the following command: On your computer's keyboard press f5 or run it with live server
+
+### Run tests
+
+To run tests, run the following command: In visual studio code with live server plugin right click and run live server.
+
```

Comment:
- [ ] [OPTIONAL] KIndly add run test commands for developers to use. This makes your README file professional. 😃 
Example: `npx hint .`, `npx stylelint "**/*.{css,scss}"`